Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -172,7 +172,7 @@ defineSuite([
 
     it('rejects readyPromise with invalid tileset.json', function() {
         var tileset = scene.primitives.add(new Cesium3DTileset({
-            url : 'invalid'
+            url : 'invalid.json'
```

Comment:
Nah, that's OK.  Nowhere near a priority for us.  If it ever comes up again we can worry about it then.